rosco-web/config/packer/install_packages.sh

```diff
@@ -16,4 +17,7 @@ if [[ "$deb_repo" != "" ]]; then
 fi
 
 sudo apt-get update
+if [[ "$upgrade" == "true" ]]; then
```

Comment:
Please surround this conditional block with blank lines to keep the formatting consistent.